apps/lfx-one/src/app/shared/components/org-selector/org-selector.component.scss

```
@@ -16,14 +16,24 @@
+  }
+
   @media (min-width: 1024px) {
     // stylelint-disable-next-line declaration-no-important
     position: fixed !important;
-    left: 344px !important;
-    top: 4px !important;
-
-    &.org-selector-panel--with-banner {
-      top: 52px !important;
-    }
+    // Right of the full sidebar (48px rail + 300px nav = 348px) with an 8px gap.
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)